docs/conf.py

```diff
@@ -0,0 +1,36 @@
+"""Configuration file for the Sphinx documentation builder.
+
+For the full list of built-in configuration values, see the documentation:
+https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+-- Project information -----------------------------------------------------
+https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+"""
+
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../src"))
+autodoc_mock_imports = ["numpy", "pandas", "croniter"]
+
+project = "coin-test"
+copyright = "2022, Olin SCOPE"
+author = "Olin SCOPE"
+release = "0.1"
+
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.napoleon",
+    "sphinx_autodoc_typehints",
+]
+
+templates_path = ["_templates"]
+exclude_patterns = []
+
+language = "python"
+
+# -- Options for HTML output -------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
+
+html_theme = "alabaster"
```

Comment:
I see that this is the default option - any rationale for choosing this over the readthedocs theme (`"sphinx_rtd_theme"`) or something with a nicer-formatted sidebar like press (`"press"`)? For any other reviewers, the list of themes can be found [here](https://sphinx-themes.org/)